src-static/update_cities_sf.R

```diff
@@ -39,7 +39,7 @@ update_cities_sf <- function(iso3) {
       pop_sf,
       adm0_a3 == iso3,
       !(name %in% c("Laayoune", "Monaco", "Singapore", "Vatican City", "Manama",
-                    "Hong Kong", "Jerusalem"))
+                    "Hong Kong", "Jerusalem", "Atlanta", "Los Angeles", "Chicago"))
```

Comment:
What's up with these cities specifically?

Reply:
There were too many across the United States, so the map was getting cluttered. Removed those that were near other cities (kept Miami instead of Atlanta, San Francisco instead of Los Angeles, and Denver instead of Chicago).

![image](https://github.com/user-attachments/assets/0cf4c587-76a6-4ad7-b10e-6b0e97cca278)

Reply:
Gotcha! Makes sense!